src/marketing/messages.js

```diff
@@ -0,0 +1,13 @@
+const DOMAIN_1 = 'https://devexpress.com/testcafe-studio';
+const DOMAIN_2 = 'https://devexpress.com/test-cafe-studio';
+const DOMAIN_3 = 'https://devexpress.com/testcafe/studio';
+
+const HOW_TO_IMPROVE_TESTING_EXPERIENCE = `Learn how to improve your testing experience with TestCafe Studio edition - ${DOMAIN_1}`;
```

Comment:
Let's add an empty line before this messages, it doesn't look good now.
![image](https://user-images.githubusercontent.com/3633477/56728191-ee0c1e80-675a-11e9-9c29-39df67c71514.png)